app/src/main/java/kz/ablazim/notesapp/create_note/CreateNoteFragment.kt

```diff
@@ -0,0 +1,27 @@
+package kz.ablazim.notesapp.create_note
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import kz.ablazim.notesapp.R
+import kz.ablazim.notesapp.databinding.FragmentCreateNoteBinding
+import kz.ablazim.notesapp.extensions.replaceFragment
+import kz.ablazim.notesapp.note_list.NoteListFragment
+
+class CreateNoteFragment : Fragment(R.layout.fragment_create_note) {
+    private val viewModel: CreateNoteViewModel by viewModels()
+    private val noteListFragment = NoteListFragment()
+    private lateinit var binding: FragmentCreateNoteBinding
```

Comment:
replace the places of this line and the 15 line. Binding should be initialized first, then you can create viewModel